rails/locales/ru.yml

```diff
@@ -0,0 +1,125 @@
+        scopes: 'Записывайте скоупы через пробел. Оставьте поле пустым для использования значений по-умолчанию'
+      edit:
+        title: 'Редактировать приложение'
+      index:
+        title: 'Ваши приложения'
+        new: 'Новое приложение'
+        name: 'Название'
+        callback_url: 'Callback URL'
+      new:
+        title: 'Новое приложение'
+      show:
+        title: 'Приложение: %{name}'
+        application_id: 'ID приложения'
+        secret: 'Secret'
+        scopes: 'Скоупы'
+        callback_urls: 'Callback urls'
+        actions: 'Действия'
+
+    authorizations:
+      buttons:
+        authorize: 'Разрешить'
+        deny: 'Отклонить'
+      error:
+        title: 'Произошла ошибка'
+      new:
+        title: 'Требуется авторизация'
+        prompt: 'Разрешить %{client_name} использовать данные Вашей учетной записи?'
+        able_to: 'Данное приложение требует следующие разрешения:'
+      show:
+        title: 'Код авторизации'
+
+    authorized_applications:
+      confirmations:
+        revoke: 'Вы уверены?'
+      buttons:
+        revoke: 'Отозвать авторизацию'
+      index:
+        title: 'Ваши приложения с авторизацией'
+        application: 'Приложение'
+        created_at: 'Создано'
+        date_format: '%Y-%m-%d %H:%M:%S'
+
+    errors:
+      messages:
+        # Common error messages
+        invalid_request: 'В запросе отсутствуют нужные параметры или он некорректен'
+        invalid_redirect_uri: 'Неправильный URI для перенаправления'
+        unauthorized_client: 'Клиент не авторизован для выполнения данного запроса'
+        access_denied: 'Доступ запрещен: сервер или владелец ресурса отклонил запрос'
+        invalid_scope: 'Неверный scope'
+        server_error: 'На авторизационном сервере произошла ошибка; запрос не выполнен'
+        temporarily_unavailable: 'В настоящее время авторизационный сервер не может обработать запрос в силу высокой загруженности или тех. работ'
+
+        #configuration error messages
+        credential_flow_not_configured: 'Resource Owner Password Credentials не сконфигурированы, необходимо настроить Doorkeeper.configure.resource_owner_from_credentials'
+        resource_owner_authenticator_not_configured: 'Resource Owner не найден, необходима конфигурация Doorkeeper.configure.resource_owner_authenticator'
+
+        # Access grant errors
+        unsupported_response_type: 'Сервер авторизации не поддерживает данный тип запроса'
+
+        # Access token errors
+        invalid_client: 'Аутентификация клиента неудачна: неверный токен, неизвестный клиент или неподдерживаемый метод аутентификации'
+        invalid_grant: 'Право на авторизацию истекло или отозвано'
+        unsupported_grant_type: 'Авторизационный сервер не поддерживает данный способ авторизации'
+
+        # Password Access token errors
+        invalid_resource_owner: 'Данные Resource Owner неверны или не найдены'
```

Comment:
It can be removed as it not used (see https://github.com/doorkeeper-gem/doorkeeper/commit/e24e991756e122c2e17978e07ba7e643b4fc4adb#diff-37c9b835927e3c363588f172de9bb6faL98)